src/media/css/feed/websites.styl

```diff
@@ -0,0 +1,110 @@
+/*
+    Stylesheet for MOW-related items in the feed.
+*/
+
+@import '../lib';
+
+.feed-item-website-tabs {
+    padding-top: 100px;
+    position: relative;
+    li {
+        display: block;
+        padding: 0;
+        position: static;
+        a {
+            background: #FFBA00;
+        }
+        &:first-child a {
+            background: #375088;
+        }
+        &:last-child a {
+            background: #CE3D3D;
+        }
+    }
+}
+
+.feed-item-website-tabs-tab {
+    align-items: center;
+    display: flex;
+    height: 102px;
+    justify-content: center;
+    left: 50%;
+    margin-left: -17%;
+    position: absolute;
+    top: 0;
+    width: 34%
+    z-index: 3;
```

Comment:
I usually recommend z-indexing in multiples of 5 or 10 in case we wish to add other elements in between at some point.